content/components/action-menu.mdx

```diff
@@ -8,40 +8,65 @@ import {Box, Button, Heading, Link} from '@primer/react'
 import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 
-## Usage
+<Note>Multi level is not available yet in the React implementation.</Note>
 
-Action menu is used for actions, navigation, to display secondary options, or single/multi select lists. They appear when users interact with buttons, actions, or other controls.
+## Usage
 
-## Anatomy
+An action menu is a list of items, with each item representing an action, command, or current selection, which can be a single or multi-select.
 
 <img
   width="960"
-  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
-  src="https://user-images.githubusercontent.com/18661030/226487158-06c6cc38-65de-438c-8028-373862fa0dfe.png"
+  alt=""
+  src="https://user-images.githubusercontent.com/980622/233374605-cee72894-fc5e-46f4-a1ea-e82aa26084ef.png"
 />
 
-## Options
+There are different ways to use menus, such as:
 
-<img
-  src="https://user-images.githubusercontent.com/18661030/226486461-c43f75c5-d9ed-415f-907c-9d2dbf71e3a2.png"
-  width="960"
-  alt=""
-/>
-<Caption>
-  The trigger button can be a standard or icon button. For single select menus that display the selection in the
-  trigger, a label must be persist either internally or externally.
-</Caption>
+- **Button**: Shows items directly related to the current view.
+- **Button (Selection)**: Shows items directly related to the button, with a title prefix, like "Sort by: Title."
+- **Context menu**: Often contains multiple levels and tends to be only available through right clicking items or through [icon buttons](/components/icon-button) that appear on hover.
+
+It's important to note that each menu item should have consistent and limited interactions:
+
+- `enter`, `click` or `tap`: Triggers an action or navigates into a submenu
+- `arrow left`, `arrow up`, `arrow down`, `arrow right`: Navigates the menu
+
+Menus are a commonly used feature across multiple platforms, and both [Apple](https://developer.apple.com/design/human-interface-guidelines/components/menus-and-actions/menus) and [Microsoft](<https://learn.microsoft.com/en-us/previous-versions/windows/desktop/bb226797(v=vs.85)>) have detailed documentation on their menus. To create a pleasant user experience, it's important to follow their expected behavior closely.
+
+## Anatomy
 
 <img
-  src="https://user-images.githubusercontent.com/18661030/226486463-893231bb-8f57-4255-83a4-7d468b2be5ed.png"
   width="960"
-  alt=""
+  alt="Action menu opened with a label for the trigger (button), overlay (menu), and the internal action list."
```

Comment:
Was there a decision made elsewhere to use `...` to indicate opening a dialog? This is a new pattern to me.

<img width="354" alt="screenshot of action menu with a danger button that opens a dialog" src="https://user-images.githubusercontent.com/18661030/234135298-a2130973-0a94-4979-8f15-29e9776c12e1.png">

Reply:
This came out of a earlier conversation you shared where it's confusing if a danger action will trigger a confirmation dialog or not. 

Both Apple and Microsoft indicate that a dialog will appear by showing a `...` at the back of the title. Therefore I think it's good practice to mimic this behaviour. 

<img width="630" alt="Screenshot 2023-04-25 at 10 37 54" src="https://user-images.githubusercontent.com/980622/234237388-9df09478-3472-4827-8c3e-e36105250243.png">

Reply:
It does seem to be a pattern used by macOS and Windows (maybe Linux too?), but I didn't realize "..." indicates a the menu item opens dialog until Max told me. It's not obvious - but I think it's ok to follow this pattern as long as it's not critical for users to know that a dialog will be opened.

Reply:
I feel like it can easily look like truncated text 😅 but I think it's interesting and we should try it.